### ISSUE TITLE
match container-name field in security policy

### ISSUE THREAD

**Issue by Ankurk99** [kubearmor/KubeArmor#885]
## Bug Report

**General Information**

In containerized un-orchestrated mode, currently we are checking only the Spec.Selector.MatchLabels field for the container name. But we should instead check for some specific field under MatchLabels (kubearmor.io/container.name) to match the container name.

**Current Behavior**
Suppose a sample policy contains:
```
  selector:
    matchLabels:
      abcd: nginx
```
then also KubeArmor will match nginx as the container name.

**Expected Behavior**
Only match container name with kubearmor.io/container.name field value 
```
  selector:
    matchLabels:
      kubearmor.io/container.name: nginx
```
Ref: https://github.com/kubearmor/KubeArmor/blob/main/KubeArmor/core/kubeUpdate.go#L1286

**Comment by HariVamsiK** [comment]
Hi
I want to work on this issue. Could you please assign it to me, it would be beneficial for my academics too.
Thanks

**Comment by Ankurk99** [comment]
Hello @HariVamsiK glad to see your interest. 
Sorry but this issue is already addressed in #886.
There are many other open issues, you might want to take a look at them. We would be more than happy to help.

**Comment by DelusionalOptimist** [comment]
@Ankurk99 since this has been handled in #886, let's close the issue?
